docs/UserGuide.md

```diff
@@ -342,10 +366,13 @@ Examples:
 
 Picture example:
 
-Before
-![Edit Event Behaviour before](images/ug-images/edit-event-behaviour-before.png)
-After
-![Edit Event Behaviour after](images/ug-images/edit-event-behaviour-after.png)
+1. Entering edit command "edit -e 1 ap/1,2 rp/1,2" to edit the participants of CS2103 Meeting.
+
+![Edit Event Behaviour before](images/ug-images/eventBehaviourImages/edit-event-before-behaviour.png)
+
+2. Event participants successfully edited.
+
+![Edit Event Behaviour after](images/ug-images/eventBehaviourImages/edit-event-behaviour-after.png)
```

Comment:
naming is a bit weird... one is `before-behaviour` and the other `behaviour-after`